client/public/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "short_name": "React App",
+  "short_name": "FightPandemics",
   "name": "Create React App Sample",
```

Comment:
and "name" ?

Reply:
https://github.com/FightPandemics/FightPandemics/issues/442#issuecomment-632919293

Reply:
updated